containers/java-postgres/test-project/test-utils.sh

```diff
@@ -0,0 +1,149 @@
+#!/bin/bash
+SCRIPT_FOLDER="$(cd "$(dirname $0)" && pwd)"
+USERNAME=${1:-vscode}
+
+if [ -z $HOME ]; then
+    HOME="/root"
+fi
+
+FAILED=()
+
+echoStderr()
+{
```

Comment:
Nit: for consistency, it seems like you are following the `{` format with no newline:

```
function() {

}
```

Reply:
`test-utils.sh` is a 'shared' file that's the same in all containers, so we've not made any edits to it

Reply:
gotcha!! Thank you.